hydra-nl/process_hydranl.py

```diff
@@ -0,0 +1,127 @@
+            for root, dirs, files in os.walk(os.path.join(hydra_werkmap, folder)):
+                    for file in files:
+
+                        # based on invoer.hyd the first columns of output dictionairy are filled
+                        if file.endswith('invoer.hyd'):
+                            location_invoer = os.path.join(root, file)
+                            invoer, terugkeertijden = lees_invoerbestand(location_invoer)
+                            
+                            # the entire path of profile is included in invoer.hyd, strip it to profile-name
+                            profielpath = invoer['algemeen']['PROFIEL']
+                            valueprofiel = profielpath.split("\\")[-1]
+                            valueprofiel = valueprofiel[:-1]
+                            value = valueprofiel.split(".")[0]
+
+                            # append first columns of output dictionairy based on invoer.hyd
+                            hydra_results['OKADER VakId'].append(value)
+                            hydra_results['q [m3/s/m]'].append(invoer['criterium']['QCR'])
+                            hydra_results['Uitvoerbestand'].append(invoer['algemeen']['UITVOERBESTAND'])
+                            hydra_results['ZSS-scenario'].append(invoer['algemeen']['SCENARIONAAM'])
+                            hydra_results['HYD_location_name'].append(invoer['algemeen']['LOCATIE'])
+                            hydra_results['HYD_location_X'].append(invoer['algemeen']['XCOORDINAAT'])
+                            hydra_results['HYD_location_Y'].append(invoer['algemeen']['YCOORDINAAT'])
+                            hydra_results['Profiel'].append(valueprofiel)
+                            
+                            # get ondergrens doorsnede-eis for a specific vak from excel
+                            for i in range(len(okader_norm_csv['VAK'])):
+                                if okader_norm_csv['VAK'].loc[i] == int(value):
+                                    req_terugkeertijd = okader_norm_csv['T_eis_hbn'].loc[i]
+
+
+                        # based on uitvoer.html (bij meest waarschijnlijke combinatie van illustratiepunt) the other columns of our dictionairy can be filled
+                        elif file.endswith('uitvoer.html'):
+                            location_uitvoer = os.path.join(root, file)
+                            tekst = lees_uitvoerhtml(location_uitvoer)
+                            cips, ip = lees_illustratiepunten(tekst)
+                            ofl = lees_ofl(tekst)
+
+                            # check if this results in 2 illustratiepunten, only one is needed in dictionairy
+                            a = (ip['bijdrage ov. freq (%)'].loc[[req_terugkeertijd][0]])
+                            if a.size > 1:
+                                rel_ip = ip.loc[[req_terugkeertijd][0]]
+                                rel_ip = rel_ip.iloc[np.argmax(rel_ip['h,teen m+NAP'])]
```

Comment:
Don't overwrite the variable. Instead use a different name or perform the operation in one line.